code/game/jobs/job_exp.dm

```diff
@@ -0,0 +1,197 @@
+
+// Globals, Defines, Etc
+
+var/global/list/exp_jobsmap = list(
+	"Crew" = list(), // all playtime from all jobs, together
+	"Command" = list(titles = command_positions),
+	"Engineering" = list(titles = engineering_positions),
+	"Security" = list(titles = security_positions),
+	"Silicon" = list(titles = nonhuman_positions),
+	"Service" = list(titles = service_positions),
+	"Medical" = list(titles = medical_positions),
+	"Science" = list(titles = science_positions),
+	"Supply" = list(titles = supply_positions),
+	"Special" = list(), // special_role
+	"Ghost" = list(), // dead/observer
+	"Exempt" = list() // special grandfather setting
+)
+
+
+// Admin Verbs
+
+
+/client/proc/cmd_admin_check_player_exp()	//Allows admins to determine who the newer players are.
+	set category = "Admin"
+	set name = "Check Player Playtime"
+	if(!check_rights(R_ADMIN))
+		return
+	var/msg = "<html><head><title>Playtime Report</title></head><BR>Playtime:<BR>"
```

Comment:
This HTML does not contain a body tag. It also does not close the html tag at any point below.